<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefRecordDlg</class>
 <widget class="QWidget" name="DlgPrefRecordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recording Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">

   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxRecordingsPath">
     <property name="title">
      <string>Recording Files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_Recordings">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel_Recordings">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>10</height>
         </size>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Directory:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>LineEditRecordings</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LineEditRecordings">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>10</height>
         </size>
        </property>
        <property name="font">
         <font/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PushButtonBrowseRecordings">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>

      <item row="1" column="0" colspan="3">
       <layout class="QHBoxLayout" name="splittingLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="LabelSplitting">
          <property name="text">
           <string>Split recordings at</string>
          </property>
          <property name="buddy">
           <cstring>comboBoxSplitting</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSplitting"/>
        </item>
        <!--item row="1" column="4" rowspan="4"-->
        <item>
         <spacer name="horizontalSpacer_split">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="CheckBoxRecordCueFile">
        <property name="text">
         <string>Create a CUE file</string>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxFileformat">
     <property name="title">
      <string>Output File Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">

      <item row="1" column="0">
       <widget class="QLabel" name="LabelLossless">
        <property name="text">
         <string>Lossless</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QHBoxLayout" name="LosslessEncLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
       </layout>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="LabelLossy">
        <property name="text">
         <string>Lossy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <layout class="QHBoxLayout" name="LossyEncLayout"/>
      </item>

      <item row="3" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>

      <item row="4" column="0">
       <widget class="QLabel" name="labelOptionGroup">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <layout class="QHBoxLayout" name="OptionGroupsLayout"/>
      </item>

      <item row="5" column="0">
       <widget class="QLabel" name="LabelQuality">
        <property name="text">
         <string>Quality</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="SliderQuality">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>11</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="TextQuality">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Quality</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>SliderQuality</cstring>
        </property>
       </widget>
      </item>

      <item row="6" column="0">
       <widget class="QLabel" name="LabelCompression">
        <property name="text">
         <string>Compression Level</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSlider" name="SliderCompression">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="TextCompression">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Compression</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>SliderCompression</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxTags">
     <property name="title">
      <string>Tags</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="LabelTitle">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>LineEditTitle</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LineEditTitle"/>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="LabelAuthor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Author</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>LineEditAuthor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LineEditAuthor"/>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="LabelAlbum">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Album</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>LineEditAlbum</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LineEditAlbum"/>
      </item>
     </layout>
    </widget>
   </item>

   <item row="7" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>LineEditRecordings</tabstop>
  <tabstop>PushButtonBrowseRecordings</tabstop>
  <tabstop>comboBoxSplitting</tabstop>
  <tabstop>CheckBoxRecordCueFile</tabstop>
  <tabstop>SliderCompression</tabstop>
  <tabstop>SliderQuality</tabstop>
  <tabstop>LineEditTitle</tabstop>
  <tabstop>LineEditAuthor</tabstop>
  <tabstop>LineEditAlbum</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
